### PR TITLE
Add Tezos docs

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

File: docs/operations/tezos/modes.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Modes
+
+Chainstack supports deploying a Tezos node on the mainnet in the following modes:
+
+* Full node — a node that stores minimal data required to reconstruct the complete chain state since the genesis.
+* Archive node — a node that stores the complete chain data and state since the genesis.
+
+To be a part of the Tezos mainnet, you can deploy either a full node or an archive node.
+
+With a full node, you can query any block information or operation but you cannot query the balances or staking rights before the current checkpoint.
```
Comment (neewy) [inline docs/operations/tezos/modes.md:18]: Should we define the current or it's self-explanatory?

File: docs/operations/tezos/tools.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+1. Originate your contract through a Chainstack-deployed Tezos node:
+
+``` sh
+SmartPy.sh originate-contract --code CONTRACT --storage STORAGE --rpc ENDPOINT
+```
+
+where
+
+* ENDPOINT — your node HTTPS endpoint.
+* CONTRACT — your complied contract.
```
Comment (neewy) [inline docs/operations/tezos/tools.md:135]:
```suggestion
* CONTRACT — your compiled contract.
```

Comment (neewy) [reply docs/operations/tezos/tools.md:135]: btw what is a compiled contract?

File: docs/tutorials/tezos/simple-fund-contract-in-ligo.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+    content: Learn how to develop in LIGO and originate a decentralized fund smart contract on the Tezos network.
+  - name: keywords
+    content: tutorial tezos ligo client deploy originate
+---
+
+# A simple fund contract in LIGO
+
+This tutorial guides you through developing, originating, and interacting with a simple decentralized fund smart contract on Tezos.
+
+The originating and the interaction with the contract for tutorial purposes is done on Florencenet, which is a testnet.
```
Comment (neewy) [inline docs/tutorials/tezos/simple-fund-contract-in-ligo.md:13]:
```suggestion
The origination and the interaction with the contract for tutorial purposes is done on Florencenet, which is a testnet.
```

?

File: docs/tutorials/tezos/simple-fund-contract-in-ligo.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+
+The simple fund contract does the following:
+
+* The contract can be funded in any amount of tez by anyone on the network.
```
Comment (neewy) [inline docs/tutorials/tezos/simple-fund-contract-in-ligo.md:17]:
```suggestion
* The contract can be funded in any amount of TEZ by anyone on the network.
```
Should we capitalize tez token mentions?

File: docs/tutorials/tezos/simple-fund-contract-in-ligo.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+
+Example:
+
+``` sh
+tezos-client --endpoint https://nd-123-456-789.p2pify.com/3c6e0b8a9c15224a8228b9a98ca1531d config update
+```
+
+### Fund your account with testnet tez
+
+The [Tezos faucet](https://faucet.tzalpha.net/) generates one account a time with test tez on all testnets at once, including Florencenet.
```
Comment (neewy) [inline docs/tutorials/tezos/simple-fund-contract-in-ligo.md:81]: at a time?

**Review by neewy — CHANGES_REQUESTED** [review]
Just a single typo — otherwise let me know what you think about the rest of comments

File: docs/operations/tezos/tools.md (hunk 2 above)
Comment (akegaviar) [reply docs/operations/tezos/tools.md:135]: Good one!

A contract written in one of the supported languages (eg LIGO or Python in this example) and then compiled to Michelson, which is Tezos' version of bytecode

File: docs/tutorials/tezos/simple-fund-contract-in-ligo.md (hunk 4 above)
Comment (akegaviar) [reply docs/tutorials/tezos/simple-fund-contract-in-ligo.md:17]: it's tez as currency similar to ether.

File: docs/tutorials/tezos/simple-fund-contract-in-ligo.md (hunk 5 above)
Comment (akegaviar) [reply docs/tutorials/tezos/simple-fund-contract-in-ligo.md]: Fixed

**Review by neewy — DISMISSED** [review]

File: docs/operations/tezos/tools.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+</template>
+</CodeSwitcher>
+
+3. Run the Tezos [client commands](https://tezos.gitlab.io/shell/cli-commands.html).
+
+Example to get the balance of an address in tez:
+
+``` sh
+$ tezos-client get balance for tz1bEQQxao8bEbvuXgh8vnSQkPJaoUvyomMP
```
Comment (easeev) [inline docs/operations/tezos/tools.md:50]:
```suggestion
tezos-client get balance for tz1bEQQxao8bEbvuXgh8vnSQkPJaoUvyomMP
```
?

File: docs/platform/view-node-resources-allocation.md [hunk 7]
```diff
@@ -19,6 +19,26 @@ To view the resources allocated to your nodes:
 
 ## Understand node resources allocation
 
+Ethereum:
+
+* Node — the CPU, memory, storage, and ancient storage allocated to your Ethereum node. The ancient storage is for the Geth freezer database to separately keep the ledger data older than three epochs. See also [an Ethereum blog post](https://blog.ethereum.org/2019/07/10/geth-v1-9-0/) for the detailed freezer explanation.
+
+Polygon PoS:
+
+* Node — the CPU, memory, and storage allocated to your Polygon PoS node.
```
Comment (easeev) [inline docs/platform/view-node-resources-allocation.md:28]: Also has ancient

File: docs/platform/view-node-resources-allocation.md [hunk 8]
```diff
@@ -19,6 +19,26 @@ To view the resources allocated to your nodes:
...
+
+Binance Smart Chain:
+
+* Node — the CPU, memory, and storage allocated to your Binance Smart Chain node.
```
Comment (easeev) [inline docs/platform/view-node-resources-allocation.md:32]: Also has ancient

File: docs/platform/view-node-resources-allocation.md [hunk 9]
```diff
@@ -19,6 +19,26 @@ To view the resources allocated to your nodes:
 
 ## Understand node resources allocation
 
+Ethereum:
+
+* Node — the CPU, memory, storage, and ancient storage allocated to your Ethereum node. The ancient storage is for the Geth freezer database to separately keep the ledger data older than three epochs. See also [an Ethereum blog post](https://blog.ethereum.org/2019/07/10/geth-v1-9-0/) for the detailed freezer explanation.
```
Comment (easeev) [inline docs/platform/view-node-resources-allocation.md:24]: Maybe we should specify somewhere that we don't charge for ancient storage

File: docs/operations/tezos/tools.md (hunk 6 above)
Comment (akegaviar) [reply docs/operations/tezos/tools.md:50]: No, there's the balance printed just below

File: docs/platform/view-node-resources-allocation.md (hunk 9 above)
Comment (akegaviar) [reply docs/platform/view-node-resources-allocation.md:24]: Done

File: docs/platform/view-node-resources-allocation.md (hunk 8 above)
Comment (akegaviar) [reply docs/platform/view-node-resources-allocation.md]: Done

File: docs/platform/view-node-resources-allocation.md (hunk 7 above)
Comment (akegaviar) [reply docs/platform/view-node-resources-allocation.md]: Done

File: docs/operations/tezos/modes.md (hunk 1 above)
Comment (akegaviar) [reply docs/operations/tezos/modes.md]: Good one, fixed

**Pull request merged**